unit/testing-utils/get_goto_model_from_c.cpp

```diff
@@ -0,0 +1,105 @@
+/*******************************************************************\
+
+Module: Get goto model
+
+Author: Daniel Poetzl
+
+\*******************************************************************/
+
+#include "get_goto_model_from_c.h"
+
+#include <ansi-c/ansi_c_language.h>
+
+#include <cbmc/cbmc_parse_options.h>
+
+#include <goto-programs/goto_convert_functions.h>
+
+#include <langapi/language_file.h>
+#include <langapi/mode.h>
+
+#include <util/cmdline.h>
+#include <util/config.h>
+#include <util/invariant.h>
+#include <util/message.h>
+#include <util/options.h>
+#include <util/symbol_table.h>
+
+#include <testing-utils/message.h>
+
+namespace
+{
+bool initialized = false;
+optionst options;
+
+void initialize()
+{
+  PRECONDITION(!initialized);
+
+  register_language(new_ansi_c_language);
+
+  cmdlinet cmdline;
+
+  config.main = std::string("main");
+  config.set(cmdline);
+
+  optionst options;
+  cbmc_parse_optionst::set_default_options(options);
```

Comment:
This doesn’t actually do anything though? `set_default_options` just sets default values on the `options`, which you then discard.

Reply:
Fixed (now using the global)